src/main/java/org/terasology/namegenerator/creature/CreatureAssetTheme.java

```diff
@@ -16,13 +16,13 @@
 
 package org.terasology.namegenerator.creature;
 
-import java.util.Collections;
-import java.util.List;
-
 import org.terasology.asset.Assets;
 import org.terasology.entitySystem.prefab.Prefab;
 import org.terasology.namegenerator.data.NameGeneratorComponent;
 
+import java.util.Collections;
```

Comment:
Do we have a rule for where which import should occur? I'm just asking to avoid moving the imports with every commit up and down ...